lib/core/index.js

```diff
@@ -1,9 +1,14 @@
 var Analytics = require('./analytics');
-
 var analytics = new Analytics();
 
 analytics.require = require;
 
 analytics.VERSION = require('../../package.json').version;
-
-module.exports = analytics;
+analytics.AGREEMENT = 'Version 1.0.0\nThis is a Licence Agreement (the "Agreement") for software modules owned by resin.io (Resin) ' +
```

Comment:
Should we get the version from `package.json`? Otherwise we will forget to update it here in the future.

Reply:
nice catch!